internal/pkg/cli/flag.go

```diff
@@ -192,4 +199,8 @@ Tasks with the same group name share the same set of resources.
 	secretAccessKeyFlagDescription = "Optional. An AWS secret access key."
 	sessionTokenFlagDescription    = "Optional. An AWS session token for temporary credentials."
 	envRegionTokenFlagDescription  = "Optional. An AWS region where the environment will be created."
+
+	retriesFlagDescription  = "Optional. The number of times to try restarting the job on a failure."
+	timeoutFlagDescription  = "Optional. The total execution time for the task, including retries."
+	scheduleFlagDescription = "The schedule on which to run this job. Can be specified using rate or cron syntax. (e.g. `20 minutes`, `@DAILY`. `0 * * * * *`)"
```

Comment:
nit: so that it renders nicely in the terminal (<80 chars). Also should we remove latin abbreviations (I'm not sure if an international audience will be familiar with it)?
```suggestion
	scheduleFlagDescription = `The schedule on which to run this job. 
Can be specified using rate or cron syntax. 
For example: "20 minutes", "@DAILY", "0 * * * * *"`
```